neqo-transport/src/cc/classic_cc.rs

```diff
@@ -469,16 +469,13 @@ impl<T: WindowAdjustment> ClassicCongestionControl<T> {
         }
     }
 
-    fn detect_persistent_congestion<'a, I>(
+    fn detect_persistent_congestion<'a>(
         &mut self,
         first_rtt_sample_time: Option<Instant>,
         prev_largest_acked_sent: Option<Instant>,
         pto: Duration,
-        lost_packets: I,
-    ) -> bool
-    where
-        I: Iterator<Item = &'a SentPacket>,
-    {
+        lost_packets: impl IntoIterator<Item = &'a SentPacket>,
```

Comment:
Using `IntoIterator` instead of `Iterator` allows other code to still pass a `Vec`, thus reducing the diff of the pull request.